hw04/BarbieWalletBalance.cs

```diff
@@ -4,6 +4,7 @@
 
 public class BarbieBank : MonoBehaviour
 {
+    
 
     // Start is called before the first frame update
 
```

Comment:
We will need to inherit from `BarbieWorld.cs`. We write the name of the class and then the name of the method followed by the brackets. 

```
GenericClass<int> myClass = new GenericClass<int>();
```

or for this problem:

```
BarbieWorld<string> barbieWorld = new BarbieWorld<string>();
barbieWorld.BarbieCareers("FashionModel");
```